### PR TITLE
release connectapi 0.3.0

### LINKED ISSUE #298: Release connectapi 0.3.0
Prepare for release:

* [x] `git pull`
* [x] Check [current CRAN check results](https://cran.rstudio.org/web/checks/check_results_connectapi.html)
* [x] Check if any deprecation processes should be advanced, as described in [Gradual deprecation](https://lifecycle.r-lib.org/articles/communicate.html#gradual-deprecation)
* [x] [Polish NEWS](https://style.tidyverse.org/news.html#news-release)
* [x] `urlchecker::url_check()` — fails as it checks the URLs even in non-executed R code, and we have the URL `http://myproxy.example.com` in the `customize-http.Rmd` vignette.
* [x] `devtools::build_readme()`
* [x] `devtools::check(remote = TRUE, manual = TRUE)`
* [x] `devtools::check_win_devel()` — waiting for email
* [x] `revdepcheck::cloud_check()` — `revdepcheck` is deprecated
* [x] Update `cran-comments.md`
* [x] `git push`

Submit to CRAN:

* [x] `usethis::use_version('minor')`
* [x] `devtools::submit_cran()` (the `devtools` docs say to use `release()`
* [x] Approve email

Wait for CRAN...

* [ ] Accepted :tada:
* [ ] `usethis::use_github_release()`
* [ ] `usethis::use_dev_version(push = TRUE)`

### PR DESCRIPTION
**Pull request by toph-allen**
This branch contains the release changes for version 0.3.0. It will merge when the release process has completed.

Closes #298.

### CODE REVIEW

**Comment by toph-allen** [comment]
v0.3.0 is released: https://github.com/rstudio/connectapi/releases/tag/v0.3.0

File: vignettes/customize-http.Rmd [hunk 1]
```diff
@@ -12,8 +12,7 @@ vignette: >
 ```{r setup, include = FALSE}
 knitr::opts_chunk$set(
   collapse = TRUE,
-  comment = "#>",
-  eval = FALSE
+  comment = "#>"
 )
 ```
```
Comment (nealrichardson) [inline vignettes/customize-http.Rmd:17]: You probably don't need this at all right? There's no R code running anymore

**Review by nealrichardson — APPROVED** [review]
One suggestion for further cleanup but otherwise LGTM, thanks for pushing this through.

File: vignettes/customize-http.Rmd (hunk 1 above)
Comment (toph-allen) [reply vignettes/customize-http.Rmd:17]: Yeah, I guess this block can just go away entirely.

**Pull request merged**